src/System.Management.Automation/engine/remoting/fanin/OutOfProcTransportManager.cs

```diff
@@ -1217,6 +1219,11 @@ protected override void CleanupConnection()
 
         private void OnOutputDataReceived(object sender, DataReceivedEventArgs e)
         {
+            if (string.IsNullOrEmpty(e.Data))
```

Comment:
Please remove this code, and add this null check to line 749 above.

Reply:
What should GetMessageGuid() return in the case? Guid.Empty?

Reply:
Yes.  Ensuing data processing will check for null or bad packets and do the appropriate thing.  Thanks!